File: 0129_BookTest/0129_BookTest/Book.m
//
//  Book.m
//  0129_BookTest
//
//  Created by katniss on 2017. 1. 29..
//  Copyright © 2017년 katniss. All rights reserved.
//

#import "Book.h"

@implementation Book

//분수를 다루는 프로그램 연습
{
    int numerator;
    int denominator;
}
-(void)print
{
    NSLog(@"%i/%i", numerator, denominator);
    
}
-(void)setNumerator:(int)n
{
    numerator = n;
    
}
-(void) setDenominator:(int)d
{
    denominator = d;
    
    
    //인스턴스 변수에 접근하는 프로그램
}
- (int) numerator
{
    return numerator;
}
-(int) denominator
{
    return denominator;
    
}

@end

File: IOS SwiftStudy/0317 Swift Study/0317 Swift Study/Book.swift
//
//  Book.swift
//  0317 Swift Study
//
//  Created by katniss on 2017. 3. 17..
//  Copyright © 2017년 katniss. All rights reserved.
//

import Foundation
//import main 안해도 자동으로 인식.


//class만듦
class Book{
    var name = ""
    var genre = ""
    var author = ""
    
    func bookPrint(){
        print("name:\(name)")
        print("genre:\(genre)")
        print("author:\(author)")
    }
}

